suricata/update/main.py

```diff
@@ -1286,8 +1291,13 @@ def _main():
        os.path.exists(config.get("suricata-conf")) and \
        suricata_path and os.path.exists(suricata_path):
         logger.info("Loading %s",config.get("suricata-conf"))
-        suriconf = suricata.update.engine.Configuration.load(
-            config.get("suricata-conf"), suricata_path=suricata_path)
+        try:
+            suriconf = suricata.update.engine.Configuration.load(
+                config.get("suricata-conf"), suricata_path=suricata_path)
+        except subprocess.CalledProcessError:
+            logger.error("Permission denied. Please alter the permissions for: %s",
+                         config.get("suricata-conf"))
+            return 1
```

Comment:
Hmm here I see two error messages.
```
26/5/2019 -- 13:07:52 - <Error> -- [ERRCODE: SC_ERR_FATAL(171)] - failed to open file: /etc/suricata/suricata.yaml: Permission denied
26/5/2019 -- 13:07:52 - <Error> -- Permission denied. Please alter the permissions for: /etc/suricata/suricata.yaml
```
This needs to be looked into.

Reply:
Here, if someone encounters with `suricata.yaml` error, the error message in the log comes automatically from `subprocess.check_output` given in `engine.Configuration.load`. I think logging the error message again here is not needed at all. The following is enough to give a description of the error.
```
27/5/2019 -- 13:30:16 - <Error> -- [ERRCODE: SC_ERR_FATAL(171)] - failed to open file: /etc/suricata/suricata.yaml: Permission denied
```
It should be sufficient to catch the exception and return. What are your thoughts on it @shivan1b ?

Reply:
Sounds well.